base/range.jl

```diff
@@ -86,6 +117,19 @@ julia> range(1, 10, length=101)
 
 julia> range(1, 100, step=5)
 1:5:96
+
+julia> try range(1, 5) catch e println(e) end
+ArgumentError("At least one of `length` or `step` must be specified")
```

Comment:
Alongside with the previous one comment, this could just be

```julia
# Must specify either length or step
julia> range(1, 5)
ERROR: ArgumentError: At least one of `length` or `step` must be specified
```

And the doctest only match the starts (if I get it correctly). For example, in OffsetArrays: https://github.com/JuliaArrays/OffsetArrays.jl/blob/87666ee004664282370122ba68e7467c13988b38/src/axes.jl#L12-L24

Reference: https://juliadocs.github.io/Documenter.jl/dev/man/doctests/#Exceptions